### PR TITLE
Some UI Fixes

### PR DESCRIPTION
**Pull request by chaitrali-r**
Some UI Fixes were required in the create vc template and create entity component

### CODE REVIEW

File: src/app/admin/create-vc-template/create-vc-template.component.html [hunk 1]
```diff
@@ -7,47 +7,44 @@
...
+          <div class="thmbCol" *ngFor="let item of credTemp; let i=index" > 
+            <div class="wrap">
               <iframe id="iframe{{i}}" class="iframe-container embed-responsive-item iframeCert" ></iframe>
             </div>
             <div class="l12 fw-bold">{{item?.title}}</div>  
           </div> 
-
-         
-      </div>
+
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-vc-template/create-vc-template.component.html:20]: @chaitrali-r Why you changed the Add template card position? 
cc @suvarnakale check its correct position?

Comment (chaitrali-r) [reply src/app/admin/create-vc-template/create-vc-template.component.html:20]: I made the change according to figma file

Comment (Pratikshakhandagale) [reply src/app/admin/create-vc-template/create-vc-template.component.html:20]: As per last discussion, we are showing it at first position. discuss with @suvarnakale she worked on this changes.

Comment (chaitrali-r) [reply src/app/admin/create-vc-template/create-vc-template.component.html]: Yes changed. thanks

Comment (suvarnakale) [reply src/app/admin/create-vc-template/create-vc-template.component.html]: yes... discussing with team we have decided its position at first

**Pull request merged**